### ISSUE TITLE
Move all demo repositories to jszgb

### ISSUE THREAD

**Issue by silvenon** [jszgb/jszgb.github.io#20]
I often do live coding presentations and therefore have repositories lying around on my account which might not be there forever. The best place for them would probably be in the jszgb GH organization, and we could do that with future demos as well. What do you think?

**Comment by dinodsaurus** [comment]
Yep, sounds good to me, but we just need to find a way how to organise it properly
